### PR TITLE
Widgets: Enable marquee for widget elements

### PR DESCRIPTION
**Pull request by rubenpingol-xibo**
relates to xibosignageltd/xibo-private#438

### CODE REVIEW

File: ui/src/core/forms.js [hunk 1]
```diff
@@ -1878,9 +1878,6 @@ window.forms = {
 
       // Add the options
       $.each(effects, function(_index, element) {
-        if (effectsType !== 'all' && element.group !== effectsType) {
```
Comment (dasgarner) [inline ui/src/core/forms.js:1881]: I thought this condition was here so that the `variant` of any effects property filtered the effects returned.

Comment (rubenpingol-xibo) [reply ui/src/core/forms.js:1881]: Yes, this condition should filter the effects based on the `variant` but having this line prevents us from showing marquee effects as previously we were setting the effects field manually for elements.

`properties-panel.js`
image

Comment (rubenpingol-xibo) [reply ui/src/core/forms.js:1881]: Should we change it's `variant = all`?

Comment (dasgarner) [reply ui/src/core/forms.js:1881]: Sorry I don't understand - does variant filtering still work without this condition (via some other mechanism)?

Comment (rubenpingol-xibo) [reply ui/src/core/forms.js:1881]: No, it won't work anymore without this condition. However, this condition prevents showing marquee effects.

**Review by maurofmferrao — APPROVED** [review]

File: ui/src/core/forms.js (hunk 1 above)
Comment (dasgarner) [reply ui/src/core/forms.js:1881]: Breaking something else can't be the solution, so perhaps the variant logic needs some refactoring. We have to be able to control which effects are returned for the property - either all, paged only or non-paged only (marquee). Please can you discuss with Mauro?

Comment (rubenpingol-xibo) [reply ui/src/core/forms.js:1881]: Yes, will do.

**Review by maurofmferrao — APPROVED** [review]

**Review by dasgarner — APPROVED** [review]

**Pull request merged**